modules/ref-dashboard-configuration-options.adoc

```diff
@@ -33,6 +58,18 @@ disableKServeAuth` | `false` | Enables the ability to use authentication with KS
 | `dashboardConfig:
 disableKServeMetrics` | `false` | Enables the ability to view KServe metrics. To disable this ability, set the value to `true`.
 | `dashboardConfig:
+disableKServeRaw` | `false` | On the *Settings → Cluster settings* page, in the *Single-model serving platform* section, shows the *Default deployment mode* list. 
+
+On the *Deploy model* dialog when using the single-model serving platform: 
+
+&nbsp; &nbsp; &nbsp; - If {org-name} OpenShift Serverless is installed, shows the *Deployment mode* list.
+
+&nbsp; &nbsp; &nbsp; - If {org-name} OpenShift Serverless is not installed, hides the *Deployment mode* list, and sets the default deployment mode to *Standard*.
+
+To hide these deployment-mode lists and unset the default deployment mode when using the single-model serving platform, set the value to `true`.
+| `dashboardConfig:
+disableModelCatalog` | `true` | Note: Do not edit the default value; this feature is not available in this version of {productname-short}.
```

Comment:
That's a funky wording -- See it but don't change it? Is it worth mentioning if we are going to say "pretend you didn't see this"?

This is another tech preview feature too (I think -- @mturley) and it's actually kinda awful for ODH usage too -- since the configmap is only part of RHOAI. This needs to get significantly lifted up by the backend team before we can really look to use this properly.

Reply:
@andrewballantyne PM decided model catalog shouldn't be documented as tech preview until 2.20, we were still finishing QA in 2.19. Same goes for ilab, which is why this text is also present above for disableFineTuning

Reply:
@andrewballantyne If the user can see dashboard config options in the UI but should not use them yet, we should document those dashboard config options to tell the users not to use them?
If we don't document those options, users might think it's an error of omission rather than that the options are deliberately not supported?

If you prefer different wording to document such items, please let me know!

Reply:
[I left another review](https://github.com/opendatahub-io/opendatahub-documentation/pull/717#pullrequestreview-2742942330) -- either remove them or document them imo.

Reply:
I removed them for now.